application.go

```diff
@@ -0,0 +1,273 @@
+package civogo
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/civo/civogo/utils"
+)
+
+// Application is the struct for the Application model
+type Application struct {
+	Name        string        `json:"name" validate:"required"`
+	ID          string        `json:"id"`
+	NetworkID   string        `json:"network_id" validate:"required"`
+	Description string        `json:"description"`
+	Image       string        `json:"image"`
+	Size        string        `json:"size"`
+	ProcessInfo []ProcessInfo `json:"process_info,omitempty"`
+	Domains     []string      `json:"domains,omitempty"`
+	SSHKeyIDs   []string      `json:"ssh_key_ids,omitempty"`
+	Config      []EnvVar      `json:"config,omitempty"`
+	// Status can be one of:
+	// - "building":  Implies platform is building
+	// - "available": Implies platform is available to accept image
+	// - "ready": Implies app is ready
+	Status           string `json:"status"`
+	CivoK3sClusterID string `json:"civo_k3s_cluster_id"`
+}
+
+// ApplicationConfig describes the parameters for a new CivoApp
+type ApplicationConfig struct {
```

Comment:
These are the only fields that should be in this struct: 

> 	Name        string   `json:"name" validate:"required" schema:"name"`
	NetworkID   string   `json:"network_id" validate:"required" schema:"network_id"`
	Description string   `json:"description" 
	Size        string   `json:"size" schema:"size"`
	SSHKeyIDs   []string `json:"ssh_key_ids"